images/runtime/dotnetcore/3.1/bullseye.Dockerfile

```diff
@@ -0,0 +1,34 @@
+ARG DEBIAN_FLAVOR
```

Comment:
I think we can delete this file and add the following to our `.gitignore`:

```
images/runtime/dotnetcore/*/bullseye.Dockerfile
```

(there's a section in the `.gitignore` for `dotnetcore` Dockerfiles)

Reply:
No changes applied to these bullseye dockerfiles, added them to gitignore but they somehow still show up.